app/src/main/java/com/google/samples/apps/sunflower/adapters/PlantDetailBindingAdapter.kt

```diff
@@ -37,9 +37,12 @@ fun imageFromUrl(view: ImageView, imageUrl: String?) {
     }
 }
 
-@BindingAdapter("goneIf")
-fun goneIf(view: FloatingActionButton, isGone: Boolean?) {
-    if (isGone == null || isGone) view.hide() else view.show()
+@BindingAdapter("minusIf")
+fun minusIf(view: FloatingActionButton, isPlanted: Boolean?) {
```

Comment:
`isPlanted` is bad naming.
*BindingAdapter*  should be independent to the app biz-domain.

Reply:
actually i dont prefer to do a bindingadapter like that.
i would try to use existing methods `app:srcCompat=@{vm.isPlanted ? R.drawable.minus : R.drawable.plus}`

Reply:
@andhie Got it, I will write the logic in xml directly.

Reply:
@XinyueZ Got it, I will rename it.

Reply:
thats why we have to use `@BindingMethod` to teach data binding about unknown attributes.
```
@BindingMethods({
        @BindingMethod(type = ImageView.class, attribute = "app:srcCompat", method = "setImageResource"),
})
```